tests/byzantium/eip196_ec_add_mul/test_ecadd.py

```diff
@@ -0,0 +1,69 @@
+"""Tests the ecadd precompiled contract."""
+
+import pytest
+from execution_testing import (
+    Alloc,
+    Environment,
+    StateTestFiller,
+    Transaction,
+)
+
+from .spec import PointG1, Spec, ref_spec_196
+
+REFERENCE_SPEC_GIT_PATH = ref_spec_196.git_path
+REFERENCE_SPEC_VERSION = ref_spec_196.version
+
+pytestmark = [
+    pytest.mark.valid_from("Byzantium"),
+    pytest.mark.parametrize("precompile_address", [Spec.ECADD], ids=[""]),
```

Comment:
Suggestion, not strictly necessary (as this info is in the module name), but this avoids a trailing "-" in the test ID.
```suggestion
    pytest.mark.parametrize("precompile_address", [Spec.ECADD], ids=["ecadd"]),
```